domains/yingxi.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "LYCaikano",
+           "email": "",
+           "discord": "1135389886657089677",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.VTZOVRZC1Hnajpqb5sSL0GaW5p8aFZsv8JIDPYQUNuUwPX7eqf0W3ZhbWhCmQubR1Cn3AmzqQbOMBz47f0bZDuqj7dPZFp1W_RstsLS1A6N11h960o3_uXvLALcwpv_12boOBJ5i_8C4lCtSLh3gZaHLbymbPVQOm76dO4cOv6XhOz0rZNd2X6sPdXqZYBFU2uJ0FGVuplRyYCOpho4JQERq3Cpf0n0y0quD39G8Wmfus61fViYr05TSLw4tJCk-Og_cDDM7E67XOU8HKVEVXBZ_dfI0XM6iLptk-Rv1UuXuhrvD1jWdhJ2FFYFn38DWVY2xGpqfFcWgiu5mvZLm6A.SH-qko0aaSLLfx6fvNbYKQ.wnwVwiupMaDQAE5U2R70p6FuzPpx782B1YInK6XypOCNKqxWPkHt28dQ9MvqUsNyKW_iakpMbOeoB9VvpKu48P00LTWFNZcINXdLPRrhDOE.MFyW5wpNHBOIvwlKpmRIwQ"
+        },
+    
+        "record": {
+            "CNAME": "selftunnel.pages.dev"
```

Comment:
I don't think that's allowed if you don't have the permission from the owners of all the website you display

Reply:
![Screenshot_20240909-130953.png](https://github.com/user-attachments/assets/2db74fe4-f4f1-4811-943e-be5f058a829c)

This is my project hosted on cloudflare